src/MqttBridgeStack.ts

```diff
@@ -7,14 +7,15 @@ import ajv = require("ajv");
 const Ajv = new ajv.default();
 
 interface Config {
-    mqttEndpoint: string;
+    nrfCloudMqttEndpoint: string;
```

Comment:
Re-ordering of parameters throughout to make them consistent, can cause some readability issues. E.g. I didn't change `mqttEndpoint` to `nrfCloudMqttEndpoint`, I moved `nrfCloudMqttEndpoint` up here in the list with the other nrfcloud params, and moved `mqttEndpoint` down with the other local params and renamed it `localMqttEndpoint` for clarity.